fe/fe-core/src/main/java/org/apache/doris/analysis/UnifiedLoadStmt.java

```diff
@@ -55,6 +42,31 @@ public void init() {
         proxyStmt.setUserInfo(getUserInfo());
     }
 
+    public static UnifiedLoadStmt buildMysqlLoadStmt(DataDescription dataDescription, Map<String, String> properties,
+                                                     String comment) {
+        StatementBase proxyStmt;
+        final ConnectContext connectContext = ConnectContext.get();
+        if (connectContext != null && connectContext.getSessionVariable().isEnableUnifiedLoad()) {
+            proxyStmt = new MysqlLoadStmt(dataDescription, properties, comment);
+        } else {
+            proxyStmt = new LoadStmt(dataDescription, properties, comment);
+        }
+        return new UnifiedLoadStmt(proxyStmt);
+    }
+
+    public static UnifiedLoadStmt buildBrokerLoadStmt(LabelName label, List<DataDescription> dataDescriptions,
+                                                      BrokerDesc brokerDesc,
+                                                      Map<String, String> properties, String comment) {
+        StatementBase proxyStmt;
+        final ConnectContext connectContext = ConnectContext.get();
+        if (connectContext != null && connectContext.getSessionVariable().isEnableUnifiedLoad()) {
+            proxyStmt = new BrokerLoadStmt(label, dataDescriptions, brokerDesc, properties, comment);
+        } else {
+            proxyStmt = new LoadStmt(label, dataDescriptions, brokerDesc, properties, comment);
+        }
+        return new UnifiedLoadStmt(proxyStmt);
+    }
+
```

Comment:
Generally LGTM
But it would be better if the code style can be optimized. Similar to this code, we can return directly without passing the else judgment again.

Reply:
done